cfme/infrastructure/pxe.py

```diff
@@ -1,16 +1,19 @@
 # -*- coding: utf-8 -*-
 """ A model of a PXE Server in CFME
 """
+import attr
+
 from navmazing import NavigateToSibling, NavigateToAttribute
 from selenium.common.exceptions import NoSuchElementException
 from widgetastic.widget import View, Text, Checkbox
 from widgetastic_manageiq import ManageIQTree, Input, ScriptBox, SummaryTable, Table
 from widgetastic_patternfly import Dropdown, Accordion, FlashMessages, BootstrapSelect, Button
 
+from cfme.base import BaseEntity, BaseCollection
 from cfme.base.login import BaseLoggedInPage
-from cfme.utils.appliance import Navigatable
-from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 from cfme.utils import conf
+from cfme.utils.appliance import get_or_create_current_appliance, Navigatable
```

Comment:
**Required**

import goes along with removal of its call below, but we're driving for the removal of get_or_create_current_appliance.

Reply:
Ignore this, since you really do need it.